cc3200/hal/prcm.c

```diff
@@ -1015,7 +1015,7 @@ void PRCMSRAMRetentionDisable(unsigned long ulSramColSel, unsigned long ulFlags)
 //!
 //! \param ulHIBWakupSrc is logical OR of valid HIB wakeup sources.
 //!
-//! This function enables individual HIB wakeup source(s). The paramter
+//! This function enables individual HIB wakeup source(s). The parameter
```

Comment:
Please don't change this file.  It's from an upstream source.  You can try getting TI to change it if you really want :)